numpy/f2py/tests/test_semicolon_split.py

```diff
@@ -0,0 +1,53 @@
+from __future__ import division, absolute_import, print_function
+
+from . import util
+from numpy.testing import assert_equal
+
+class TestMultiline(util.F2PyTest):
+    suffix = ".pyf"
+    module_name = "multiline"
+    code = """
+python module {module}
+    usercode '''
+void foo(int* x) {{ 
+    char dummy = ';';
+    *x = 42; 
+}}
+'''
+    interface
+        subroutine foo(x)
+	    intent(c) foo
+            integer intent(out) :: x
+        end subroutine foo
+    end interface
+end python module {module}
+    """.format(module=module_name)
+
+    def test_multiline(self):
+        assert_equal(self.module.foo(), 42)
+
+class TestCallstatement(util.F2PyTest):
+    suffix = ".pyf"
+    module_name = "callstatement"
+    code = """
+python module {module}
+    usercode '''
+void foo(int* x) {{
+}}
+'''
+    interface
+        subroutine foo(x)
+	    intent(c) foo
+            integer intent(out) :: x
+	    callprotoargument int*
```

Comment:
Indentation?

Reply:
Funny, that looks good in the code. Perhaps there are some hard tabs?

Reply:
Yep, you have a bunch of hard tabs scattered in the file. A `retab` should clean this up in vim.

Reply:
OK, there are other files with this problem. I'll put this in and make a fixup PR.